rest/model/admin_test.go

```diff
@@ -122,6 +122,8 @@ func TestModelConversion(t *testing.T) {
 	assert.EqualValues(testSettings.AuthConfig.Github.ClientId, utility.FromStringPtr(apiSettings.AuthConfig.Github.ClientId))
 	assert.EqualValues(testSettings.AuthConfig.Multi.ReadWrite[0], apiSettings.AuthConfig.Multi.ReadWrite[0])
 	assert.Equal(len(testSettings.AuthConfig.Github.Users), len(apiSettings.AuthConfig.Github.Users))
+	assert.EqualValues(testSettings.Bucket.LogBucket, utility.FromStringPtr(apiSettings.Bucket.LogBucket))
```

Comment:
should we also test the data level ([here](https://github.com/evergreen-ci/evergreen/blob/276f46116f490f1fda52987e5c32cc5e61ab8151/rest/data/admin_test.go#L88))? and the other direction ([here](https://github.com/evergreen-ci/evergreen/blob/2ae72ff6826e8ad21e25fd862cb3531d7e273f16/rest/model/admin_test.go#L262))?

Reply:
I added tests going in the other direction (for the cedar and data pipes configs as well). As for the test at the data level, that may be unnecessary since the test itself just does a spot check on the values—the more thorough testing is done here.